tests/Functional/lib/AutoregisterClassmapPluginTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace Functional\lib;
+
+use Chris\Composer\AutoregisterClassmapPlugin\AutoregisterClassmapPlugin;
+use Composer\Composer;
+use Composer\Config;
+use Composer\IO\IOInterface;
+use Composer\Package\RootPackage;
+use Composer\Script\Event;
+use Phake;
+use Phake_IMock;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+class AutoregisterClassmapPluginTest extends \PHPUnit_Framework_TestCase
+{
+    /**
```

Comment:
In general: Add some comments on your constants to explain from where (for example path from the root directory to ...)